Dockerfile

```diff
@@ -44,9 +44,16 @@ CMD ["-a", "build"]
 #
 # DOCKER_BUILDKIT=1 docker build --target go --tag ockam/tool/go:latest .
 # docker run --rm --volume "$(pwd):/project" ockam/tool/go:latest
-FROM golang:1.11.2-alpine3.8@sha256:692eff58ac23cafc7cb099793feb00406146d187cd3ba0226809317952a9cf37 as go
+FROM golang:1.11.2-alpine3.8@sha256:692eff58ac23cafc7cb099793feb00406146d187cd3ba0226809317952a9cf37 as go-alpine
 ENV GOOS=linux GOARCH=amd64 CGO_ENABLED=1
-RUN apk --update add --no-cache gcc g++ libtool git
+RUN apk --update add --no-cache gcc g++ libtool git libgcc
+WORKDIR /project
+ENTRYPOINT ["go"]
+
+FROM golang:1.11.5-stretch as go
+ENV GOOS=linux GOARCH=amd64 CGO_ENABLED=1
+RUN apt-get update
+RUN apt-get install gcc g++ libtool git
```

Comment:
this line needs a `apt-get install -y`, the missing `-y` makes the build fail

Reply:
https://dev.azure.com/ockam-network/ockam/_build/results?buildId=416&view=logs&jobId=12f1170f-54f2-53f3-20dd-22fc7dff55f9&taskId=9c939e41-62c2-5605-5e05-fc3554afc9f5&lineStart=699&lineEnd=700&colStart=1&colEnd=1